JapaneseFilter/sections/specific.txt

```diff
@@ -1156,7 +1155,10 @@ how-to-inc.com##.adblocks
 ||game8.jp/assets/*/ads/
 ||game8.jp/assets/pc/jack_ad/
 game8.jp###g8-ad-placement-pc_side_head
+game8.jp,nurumayu.net###js-popup
+game8.jp##.ad-skyflag-wrapper
```

Comment:
not found

Reply:
`game8.jp,nurumayu.net###js-popup`  is added in this issue https://github.com/AdguardTeam/AdguardFilters/issues/123677.  I can not reproduce.
Should I remove this rule?

`game8.jp##.ad-skyflag-wrapper`is added in this issue  https://github.com/AdguardTeam/AdguardFilters/issues/124320. I can reproduc on mobile. 
Should I move this rule to mobile filter?

Reply:
Removed game8.jp from `###js-popup`

Reply:
> Should I move this rule to mobile filter?

Moved.

Reply:
Please update PR.

Reply:
ok,done